rqt_plot/src/rqt_plot/plot.py

```diff
@@ -108,17 +116,28 @@ def add_arguments(parser):
             help='Start in paused state')
         group.add_argument('-e', '--empty', action='store_true', dest='start_empty',
             help='Start without restoring previous topics')
+        group.add_argument('-l', '--labels', action='append', default=[],
+                           dest='labels', help='Topic labels')
+        group.add_argument('-T', '--title', dest='title', default='',
+                           help='Window title')
         group.add_argument('topics', nargs='*', default=[], help='Topics to plot')
 
     def _update_title(self):
-        self._widget.setWindowTitle(self._data_plot.getTitle())
+        title = self._title
+        if title is None or title == '':
+            title = self._data_plot.getTitle()
```

Comment:
This was previously getting its value from self._widget.windowTitle() why did it change?